plugins/servprotect.py

```diff
@@ -2,18 +2,27 @@
 from expiringdict import ExpiringDict
 
 from pylinkirc import utils
+from pylinkirc.conf import conf
 from pylinkirc.log import log
 
-# TODO: make length and time configurable
-savecache = ExpiringDict(max_len=5, max_age_seconds=10)
-killcache = ExpiringDict(max_len=5, max_age_seconds=10)
+# check for definitions, if plugin isn't
+# loaded we stop anyways
+if conf.get('servprotect') != None:
+    length = conf['servprotect'].get('length', 5)
+    age    = conf['servprotect'].get('max_age', 10)
+else:
+    length = 5
+    age    = 10
```

Comment:
This is more efficient by saving extra dict key lookups:
```python
servprotect_conf = conf.conf.get('servprotect', {})
length = servprotect_conf.get('length,' 5)
age = servprotect_conf.get('age', 10)
```

Also, don't indent variable names to line up the values (style tips for this project).